Vowpal Wabbit - Contextual Bandits.ipynb

```diff
@@ -0,0 +1,705 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img width=\"200\" src=\"https://mmlspark.blob.core.windows.net/graphics/emails/vw-blue-dark-orange.svg\" />\n",
+    "\n",
+    "# Contextual-Bandits using Vowpal Wabbit\n",
+    "\n",
+    "[Azure Personalizer](https://azure.microsoft.com/en-us/products/cognitive-services/personalizer) emits logs in DSJSON-format. This example demonstrates how to perform off-policy evaluation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Read dataset"
```

Comment:
Maybe put these headers with Steps, and make them one level out (###) to be a bit bigger in the NB